teuthology/orchestra/remote.py

```diff
@@ -8,6 +8,7 @@
 from teuthology.orchestra import connection
 from teuthology.orchestra import console
 from teuthology.orchestra.opsys import OS
+from teuthology.provision import get_reimage_types
```

Comment:
Just do:
```
import teuthology.provision
```

Reply:
done

Reply:
reverted this change, it gave me a bizarre import error.